context/testing.go

```diff
@@ -0,0 +1,46 @@
+package context
+
+import (
+	"io"
+	"reflect"
+	"testing"
+)
+
+func StubBackupConfig() func() {
+	orig := BackupConfigFile
+	BackupConfigFile = func(_ string) error {
+		return nil
+	}
+
+	return func() {
+		BackupConfigFile = orig
+	}
+}
+
+func StubWriteConfig(w io.Writer) func() {
+	orig := WriteConfigFile
+	WriteConfigFile = func(fn string, data []byte) error {
+		_, err := w.Write(data)
+		return err
+	}
+	return func() {
+		WriteConfigFile = orig
+	}
+}
+
+func StubConfig(content string) func() {
+	orig := ReadConfigFile
+	ReadConfigFile = func(fn string) ([]byte, error) {
+		return []byte(content), nil
+	}
+	return func() {
+		ReadConfigFile = orig
+	}
+}
+
+func eq(t *testing.T, got interface{}, expected interface{}) {
```

Comment:
Instead of having to import the `testing` package in a file that doesn't end in `_test.go`, could we describe `t` as simply conforming to an interface that only the `Helper()` method? Since this is the only method of `t` that we are invoking?

Or better yet, we can move the definition of `eq` into a `_test.go` file